lib/tasks/pkg.rake

```diff
@@ -24,5 +24,20 @@ namespace :pkg do
     version = `git show #{ref}:VERSION`.chomp.chomp('-develop')
     raise "can't find VERSION from #{ref}" if version.length == 0
     `git archive --prefix=foreman-#{version}/ #{ref} | bzip2 -9 > pkg/foreman-#{version}.tar.bz2`
+
+    `npm install`
+    `node_modules/.bin/webpack --bail --config config/webpack.config.js`
```

Comment:
If we do the actual webpack build here we don't need to include the modules in the tarball at all, but it might be more correct to only include the modules directory and leave the webpack compilation for the build process.